README.md

```diff
@@ -327,12 +327,37 @@ The IP configuration supports the following options:
     [`ipv4.dhcp-send-hostname`](https://developer.gnome.org/NetworkManager/stable/nm-settings.html#nm-settings.property.ipv4.dhcp-send-hostname)
     property.
 
-* `dns`, `dns_search` and `dns_options`
+* `dns`, `dns_search`
 
     Manual DNS configuration can be specified via a list of addresses
     given in the `dns` option, a list of domains to search given in the
-    `dns_search` option and a list of dns options to set given in the
-    `dns_options`.
+    `dns_search` option. `dns_search` is only supported for IPv4 nameservers.
+
+* `dns_options`
+
+    `dns_options` is only supported for the NetworkManager provider and IPv4
+    nameservers. Manual DNS configuration via a list of DNS options can be given in the
+    `dns_options`. The list of supported DNS options is the same as described in
+    [man 5 resolv.conf](https://man7.org/linux/man-pages/man5/resolv.conf.5.html).
```

Comment:
man resolv.conf  shows more DNS options than the list below, so " The list of supported DNS options is the same as described in" is not true.

Reply:
Those extra DNS options in man resolv.conf primarily work for IPv6 nameserver, and network role does not support configuring DNS options for IPv6 nameservers at the present .

Reply:
ok, but then you cannot write that the list is the same.

Reply:
I changed into  "The list of supported DNS options for IPv4 nameservers is the same as described in [man 5 resolv.conf](https://man7.org/linux/man-pages/man5/resolv.conf.5.html)".

Reply:
@tyll, I changed into "The list of supported DNS options for IPv4 nameservers is described in [man 5 resolv.conf](https://man7.org/linux/man-pages/man5/resolv.conf.5.html)".

Reply:
Since we are planning to support dns options for all address family, maybe we should add the missed dns options for IPv6 nameservers and improve the argvalidator. Right ?